api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -215,6 +215,17 @@ message ExtAuthz {
   // Whether to increment cluster statistics (e.g. cluster.<cluster_name>.upstream_rq_*) on authorization failure.
   // Defaults to true.
   google.protobuf.BoolValue charge_cluster_response_stats = 20;
+
+  // Whether to send headers values as raw bytes (instead of a string) in authentication request.
```

Comment:
This knob now controls more than just whether the values are sent as bytes or not; it also changes how multiple headers of the same name are handled. I'm not sure what the right name is, but maybe it should be more general than the current name.

Also, I think we should document that we recommend setting this to true for all new use cases, and false is for legacy support (although it's not deprecated; we'll support both modes forever).

Reply:
Agreed. Something like "encode_raw_headers" sounds ok to me.